### PR TITLE
doc(qa): Develop a Hatchery executable test plan

### PR DESCRIPTION
**Pull request by themarcelor**
Jira Ticket: [PXP-4946](https://ctds-planx.atlassian.net/browse/PXP-4946)
Develop a Hatchery executable test plan

### CODE REVIEW

**Review by jingh8 — APPROVED** [review]

File: docs/testplans/hatchery/hatchery-executable-test-plan.md~ [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Object Management API (aka: Shepherd) test plan
+
```
Comment (frickjack) [inline docs/testplans/hatchery/hatchery-executable-test-plan.md~:2]: This is great - thanks!
I think you don't want to checkin this file - maybe add a line to `.gitignore` ...

**Review by frickjack — COMMENTED** [review]
Awesome!  Just remove the `~` file

**Review by atharvar28 — COMMENTED** [review]
Looks Beautiful 👍

**Review by atharvar28 — APPROVED** [review]

File: docs/testplans/hatchery/hatchery-executable-test-plan.md~ (hunk 1 above)
Comment (themarcelor) [reply docs/testplans/hatchery/hatchery-executable-test-plan.md~:2]: ugh, I totally missed this message.
Let me fix that...

**Pull request merged**